diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -182,8 +183,10 @@ enum class Warnings(
                    node: ASTNode,
                    canBeAutoCorrected: Boolean = this.canBeAutoCorrected,
                    autoFix: () -> Unit) {
-        warn(configRules, emit, canBeAutoCorrected, freeText, offset, node)
-        fix(configRules, isFixMode, node, autoFix)
+        if (isRuleFromActiveChapter(configRules.disabledChapters(), this)) {
```

Comment:
if you will make it an extension function - it will be much more readable: this.isRuleFromActiveChapter